<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Friends_Button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>95</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>好友</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Group_Button">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>10</y>
     <width>95</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>聊天室</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Searcg_Line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Search_Button">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>80</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QListWidget" name="Friends_listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>300</width>
     <height>440</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="Groups_listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>300</width>
     <height>440</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="Search_ListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>300</width>
     <height>440</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
